src/pages/TeamPage/TeamPage.tsx

```diff
@@ -212,7 +212,8 @@ export const TeamPage: React.FC = () => {
   ) => {
     if (!currentUser) return false;
 
-    const membersWithSameRole = teamMembers.filter((teamMember: FetchTeamMembersResponse) => {
+    let membersWithSameRole: FetchTeamMembersResponse[] = [];
+    membersWithSameRole = teamMembers.filter((teamMember: FetchTeamMembersResponse) => {
```

Comment:
No need to use let to set types
```suggestion
    const membersWithSameRole: FetchTeamMembersResponse[] = teamMembers.filter((teamMember: FetchTeamMembersResponse) => {
```